src/index/DeltaTriples.h

```diff
@@ -15,6 +15,7 @@
 #include "backports/three_way_comparison.h"
 #include "engine/LocalVocab.h"
 #include "global/IdTriple.h"
+#include "global/RuntimeParameters.h"
```

Comment:
I believe including this header is still relatively expensive. Please move it to the cpp File and hide its usage behind a `getDefaultValue` or something.

Reply:
Question: the advantage over initializing it in the member initializer list in the constructor in the `.cpp` (which sonarcloud complains about) is that we get the default value for all constructors at the cost of this additional function. Are there more advantages to doing it the way you propose.